Issue building with docker compose using official Dockerfile

### Backend impacted

Other / All

### Operating system

Linux

### Hardware

GPU with CUDA

### Description

Building with docker compose build using the official Dockerfile on NVIDIA DGX Spark.

````
#14 44.02       error: failed to run custom build command for `audiopus_sys v0.2.2`
#14 44.02 
#14 44.02       Caused by:
#14 44.02         process didn't exit successfully:
#14 44.02       `/root/.cache/uv/sdists-v9/pypi/sphn/0.1.12/ydE6YVzlj6lDsY9BzIcBr/src/target/release/build/audiopus_sys-04d4876f6c03d266/build-script-build`
#14 44.02       (exit status: 101)
#14 44.02         --- stdout
#14 44.02         cargo:info=No feature or environment variable found, linking by
#14 44.02       default.
#14 44.02         cargo:rerun-if-env-changed=OPUS_NO_PKG_CONFIG
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG_aarch64-unknown-linux-gnu
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG_aarch64_unknown_linux_gnu
#14 44.02         cargo:rerun-if-env-changed=HOST_PKG_CONFIG
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG_PATH_aarch64-unknown-linux-gnu
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG_PATH_aarch64_unknown_linux_gnu
#14 44.02         cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG_PATH
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_aarch64-unknown-linux-gnu
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_aarch64_unknown_linux_gnu
#14 44.02         cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_aarch64-unknown-linux-gnu
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_aarch64_unknown_linux_gnu
#14 44.02         cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
#14 44.02         cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
#14 44.02         cargo:info=`pkg_config` could not find `Opus`.
#14 44.02         cargo:info=Opus source path used:
#14 44.02       "/root/.cache/puccinialin/cargo/registry/src/index.crates.io-1949cf8c6b5b557f/audiopus_sys-0.2.2/opus".
#14 44.02         cargo:info=Building Opus via CMake.
#14 44.02         CMAKE_TOOLCHAIN_FILE_aarch64-unknown-linux-gnu = None
#14 44.02         CMAKE_TOOLCHAIN_FILE_aarch64_unknown_linux_gnu = None
#14 44.02         HOST_CMAKE_TOOLCHAIN_FILE = None
#14 44.02         CMAKE_TOOLCHAIN_FILE = None
#14 44.02         CMAKE_GENERATOR_aarch64-unknown-linux-gnu = None
#14 44.02         CMAKE_GENERATOR_aarch64_unknown_linux_gnu = None
#14 44.02         HOST_CMAKE_GENERATOR = None
#14 44.02         CMAKE_GENERATOR = None
#14 44.02         CMAKE_PREFIX_PATH_aarch64-unknown-linux-gnu = None
#14 44.02         CMAKE_PREFIX_PATH_aarch64_unknown_linux_gnu = None
#14 44.02         HOST_CMAKE_PREFIX_PATH = None
#14 44.02         CMAKE_PREFIX_PATH = None
#14 44.02         CMAKE_aarch64-unknown-linux-gnu = None
#14 44.02         CMAKE_aarch64_unknown_linux_gnu = None
#14 44.02         HOST_CMAKE = None
#14 44.02         CMAKE = None
#14 44.02         running: cd
#14 44.02       "/root/.cache/uv/sdists-v9/pypi/sphn/0.1.12/ydE6YVzlj6lDsY9BzIcBr/src/target/release/build/audiopus_sys-1c6216e7802ea9f9/out/build"
#14 44.02       && CMAKE_PREFIX_PATH="" LC_ALL="C" "cmake"
#14 44.02       "/root/.cache/puccinialin/cargo/registry/src/index.crates.io-1949cf8c6b5b557f/audiopus_sys-0.2.2/opus"
#14 44.02       "-DCMAKE_INSTALL_PREFIX=/root/.cache/uv/sdists-v9/pypi/sphn/0.1.12/ydE6YVzlj6lDsY9BzIcBr/src/target/release/build/audiopus_sys-1c6216e7802ea9f9/out"
#14 44.02       "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC"
#14 44.02       "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -ffunction-sections
#14 44.02       -fdata-sections -fPIC" "-DCMAKE_CXX_COMPILER=/usr/bin/c++"
#14 44.02       "-DCMAKE_ASM_FLAGS= -ffunction-sections -fdata-sections -fPIC"
#14 44.02       "-DCMAKE_ASM_COMPILER=/usr/bin/cc" "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
#14 44.02         -- Configuring incomplete, errors occurred!
#14 44.02 
#14 44.02         --- stderr
#14 44.02         CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
#14 44.02           Compatibility with CMake < 3.5 has been removed from CMake.
#14 44.02 
#14 44.02           Update the VERSION argument <min> value.  Or, use the <min>...<max>
#14 44.02       syntax
#14 44.02           to tell CMake that the project requires at least <min> but has been
#14 44.02       updated
#14 44.02           to work with policies introduced by <max> or earlier.
#14 44.02 
#14 44.02           Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring
#14 44.02       anyway.
#14 44.02 
#14 44.02 
#14 44.02 
#14 44.02         thread 'main' (1109) panicked at
#14 44.02       /root/.cache/puccinialin/cargo/registry/src/index.crates.io-1949cf8c6b5b557f/cmake-0.1.54/src/lib.rs:1119:5:
#14 44.02 
#14 44.02         command did not execute successfully, got: exit status: 1
#14 44.02 
#14 44.02         build script failed, must exit now
#14 44.02         note: run with `RUST_BACKTRACE=1` environment variable to display
#14 44.02       a backtrace
#14 44.02       warning: build failed, waiting for other jobs to finish...
#14 44.02       💥 maturin failed
#14 44.02         Caused by: Failed to build a native library through cargo
#14 44.02         Caused by: Cargo build finished with "exit status:
#14 44.02       101": `env -u CARGO PYO3_BUILD_EXTENSION_MODULE="1"
#14 44.02       PYO3_ENVIRONMENT_SIGNATURE="cpython-3.12-64bit"
#14 44.02       PYO3_PYTHON="/root/.cache/uv/builds-v0/.tmpoj1sMr/bin/python"
#14 44.02       PYTHON_SYS_EXECUTABLE="/root/.cache/uv/builds-v0/.tmpoj1sMr/bin/python"
#14 44.02       "cargo" "rustc" "--profile" "release" "--features"
#14 44.02       "pyo3/extension-module" "--message-format"
#14 44.02       "json-render-diagnostics" "--manifest-path"
#14 44.02       "/root/.cache/uv/sdists-v9/pypi/sphn/0.1.12/ydE6YVzlj6lDsY9BzIcBr/src/Cargo.toml"
#14 44.02       "--lib"`
#14 44.02       Error: command ['maturin', 'pep517', 'build-wheel', '-i',
#14 44.02       '/root/.cache/uv/builds-v0/.tmpoj1sMr/bin/python', '--compatibility',
#14 44.02       'off'] returned non-zero exit status 1
#14 44.02 
#14 44.02       hint: This usually indicates a problem with the package or the build
#14 44.02       environment.
#14 44.02   help: `sphn` (v0.1.12) was included because `moshi-personaplex` depends
#14 44.02         on `sphn`
#14 ERROR: process "/bin/sh -c uv sync" did not complete successfully: exit code: 1
------
 > [base 7/8] RUN uv sync:
44.02       "/root/.cache/uv/sdists-v9/pypi/sphn/0.1.12/ydE6YVzlj6lDsY9BzIcBr/src/Cargo.toml"
44.02       "--lib"`
44.02       Error: command ['maturin', 'pep517', 'build-wheel', '-i',
44.02       '/root/.cache/uv/builds-v0/.tmpoj1sMr/bin/python', '--compatibility',
44.02       'off'] returned non-zero exit status 1
44.02 
44.02       hint: This usually indicates a problem with the package or the build
44.02       environment.
44.02   help: `sphn` (v0.1.12) was included because `moshi-personaplex` depends
44.02         on `sphn`
------
Dockerfile:19

--------------------

  17 |     COPY moshi/ /app/moshi/

  18 |     RUN uv venv /app/moshi/.venv --python 3.12

  19 | >>> RUN uv sync

  20 |     

  21 |     RUN mkdir -p /app/ssl

--------------------

failed to solve: process "/bin/sh -c uv sync" did not complete successfully: exit code: 1
```

### Extra information

### Environment

Linux (Docker container built from nvcr.io/nvidia/cuda:12.4.1-runtime-ubuntu22.04 using docker compose build on NVIDIA DGX Spark)

- Python version: Python 3.12 (inside the container / virtual environment created during build)
- PyTorch version: Installed during image build from the official dependencies
- CUDA version: CUDA 12.4 (from the NVIDIA CUDA base image)
- GPU model and memory: NVIDIA DGX Spark GPUs

--- a/Dockerfile
+++ b/Dockerfile
@@ -8,8 +8,6 @@ COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
 RUN apt-get update && apt-get install -y --no-install-recommends \
     build-essential \
     pkg-config \
-    clang \
-    ca-certificates \
     libopus-dev \
  && rm -rf /var/lib/apt/lists/*
 

